configureLocalEnv.js

```diff
@@ -13,84 +13,84 @@ try {
   console.log(ethConfigFile)
   console.log(solConfigFile)
 
-  const remoteHost = process.env.AUDIUS_REMOTE_DEV_HOST
+  const remoteHost = import.meta.env.AUDIUS_REMOTE_DEV_HOST
```

Comment:
all replacements of `process.env` with `import.meta.env` are also a CRA->Vite side effect